packages/cacti-copm-core/src/main/proto/generated/openapi/models/asset_lock_claim_v1_pb.proto

```diff
@@ -22,16 +22,10 @@ message AssetLockClaimV1PB {
 
   string lock_id = 1;
 
-  AssetAccountV1PB source = 359634918;
-
   AssetAccountV1PB destination = 356105204;
```

Comment:
I think I forgot to ask this in previous PR? or sorry if I'm asking again, but why such high numbers here? Usually we start from 1,2,3... and so on, contiguous small numbers.
Any reason for this?

Reply:
This was discussed here: https://github.com/hyperledger-cacti/cacti/pull/3546#discussion_r1766255442 and also on the project channel on October 24. 

The proto files are generated from the open api specification. The generator makes the numbers from the hash of the variable name, so it is guaranteed to be unique within the proto message, serving its intended purpose.  As requested by @VRamakrishna I tried setting the 'x-protobuf-index' in the openapi spec, however the variable is ignored for members of type 'ref'.

Reply:
If there aren't too many such proto files, can we manually edit them to assign readable (i.e, falling in a sequence 1..2..3..) numbers? I was going to suggest this in the previous PR too, but thought it was better to merge it first as I didn't want to hold it up longer than it already had been.

Reply:
This is a non-trivial request, as there are a number of files and the manual edits would need to be re-done when the openapi spec changes.  Please note that other protobuf files in the project generated with this toolset also have large numbers, for instance the directory here: 

https://github.com/jenniferlianne/cacti/blob/copm_register_assets_in_chaincode/packages/cactus-plugin-keychain-memory/src/main/proto/generated/openapi/models/has_keychain_entry_response_v1_pb.proto   

In some ways, generating a hash from the variable name is a better approach as it removes human error.